app/code/Magento/Directory/Helper/Data.php

```diff
@@ -190,8 +190,8 @@ public function getRegionJson()
         \Magento\Framework\Profiler::start('TEST: ' . __METHOD__, ['group' => 'TEST', 'method' => __METHOD__]);
         if (!$this->_regionJson) {
             $scope = $this->getCurrentScope();
-            $scopeKey = $scope['value'] ? '_' . implode('_', $scope) : null;
-            $cacheKey = 'DIRECTORY_REGIONS_JSON_STORE' . $scopeKey;
+            $scopeKey = $scope['value'] ? '_' . implode('_', $scope) : '';
+            $cacheKey = 'DIRECTORY_REGIONS_JSON' . $scopeKey;
```

Comment:
Can you give more details on why this change here is relevant?

Reply:
Doesn't make sense to have string **_STORE** in cache key template, because `$scope['type']` already contains scope type. In result currently cache key for store takes value **DIRECTORY_REGIONS_JSON_STORE_STORE_1**.